src/Round.js

```diff
@@ -1,28 +1,42 @@
+import data from './data';
+import Player from './Player';
+import Game from './Game';
+import Wheel from './Wheel';
+
 class Round {
   constructor(game) {
     this.game = game;
+    this.round = this.game.roundCounter;
     this.puzzleBank = [];
-    this.puzzle = {};
+    this.puzzle = this.choosePuzzle();
+    this.turn = this.makeNewTurn();
+    // this.wheel = new Wheel();
   }
 
-  startNewRound() {
-    //wheel.generateCurrentWheel()
+  makeNewTurn(player) {
+    // let playerIndex = this.game.players
+    // if (this.playerIndex < this.game.players.length -1) {
+    //   playerIndex ++;
+    // } else {
+    //   let playerIndex = 0;
+    // }
+    // return new Turn(this, this.game.players[playerIndex]);
   }
-
-  //makeNewTurn()
   
-  choosePuzzle() {
-
-  }
 
   createPuzzleBank() {
-
+    let randomNum = Math.ceil(Math.random() * 23);
+    let oneWrdPzl = data.puzzles.one_word_answers.puzzle_bank[randomNum];
```

Comment:
Similarly here, `data` seems to be global to this file. Data should be passed into the method or called as a property on the class. Either way is probably fine, but aim to not have `data` global to the file.